src/l10n/nb.json

```
@@ -935,7 +935,91 @@
     },
     "votes": "Stemmer",
     "replies": "Svar",
-    "views": "Visninger"
+    "views": "Visninger",
+    "comments": {
+      "sectionTitle": "Har du spørsmål eller kommentarer til {entityType}?",
+      "entityTypes": {
+        "dataset": "datasettet",
+        "dataservice": "datatjenesten",
+        "concept": "begrepet",
+        "informationmodel": "informasjonsmodellen",
+        "public_service": "den offentlige tjenesten",
+        "event": "hendelsen"
+      },
+      "ingress": "Kommentaren blir synlig for andre brukere og besvares her.",
+      "noCommunityUser": "Lag en konto på Datalandsbyen for å kommentere",
+      "buttons": {
+        "feedback": "Still spørsmål",
+        "logInFeedback": "Logg inn for å stille spørsmål",
+        "reply": "Skriv et svar",
+        "edit": "Rediger",
+        "delete": "Slett",
+        "expandComments": "Vis alle {count} tilbakemeldinger",
+        "collapseComments": "Skjul tilbakemeldinger",
+        "expandReplies": "Vis alle {count} svar",
+        "collapseReplies": "Skjul svar"
+      }
+    },
+    "composer": {
+      "commenter": "Kommenterer som ",
```

```suggestion
      "commenter": "Kommentator som ",
```

---

Stemmer dette?

---

Kanskje, kommer an på konteksten

---

Kan være dette er bedre:
```suggestion
      "commenter": "Kommentator",
```
Men igjen, jeg kjenner ikke helt til kontekst

---

Tanken er at over feltet du skriver en kommentar skal det stå f.eks.:
`Kommenterer som NilsOveTen`